config/backend-config/backend-config.go

```diff
@@ -208,19 +210,22 @@ func newForDeployment(deploymentType deployment.Type, configEnvHandler types.Con
 			configJSONPath:   configJSONPath,
 			configBackendURL: parsedConfigBackendURL,
 			configEnvHandler: configEnvHandler,
+			cpRouterURL:      cpRouterURL,
```

Comment:
It doesn't look like we're using this one.

Reply:
Removed this too